packages/bpk-component-badge/src/BpkBadge.js

```diff
@@ -42,7 +45,15 @@ const badgeTypeClassNames = {
   [BADGE_TYPES.outline]: getClassName('bpk-badge--outline'),
 };
 
-const BpkBadge = props => {
+type Props = {
```

Comment:
We've started a convention of exporting the `type` too, so just add the `export` keyword in front of this line.

Here's an example of how that's surfaced in `index.js`: https://github.com/Skyscanner/backpack/blob/master/native/packages/react-native-bpk-component-nudger/index.js